Free users note for analytics (#3759)
wso2/docs-apim: en/docs/observe/api-manager-analytics/configure-analytics/register-for-analytics.md

# Register for Analytics

API Manager 4.0.0 offers analytics as a cloud service. Hence, users need to register with analytics cloud in order to
use API Manager Analytics

## Prerequisites

You need to be a registered Choreo user in order to utilize Analytics Cloud.
If you are a standalone user trying out the product, please signup to Choreo by visiting
[https://console.choreo.dev](https://console.choreo.dev). If you are signing up on behalf of an enterprise and need to share the Analytics Cloud among a group of people, please follow [this]({{base_path}}/observe/api-manager-analytics/configure-analytics/working-with-organizations)

> document to signup and create a group which is formally referred as an organization.

Reviewer: ```suggestion
document to signup and create a group that is formally referred as an organization.
```

## Register On-premise Environment

Please follow below steps to register your on-premise environment.

- Log on to [Choreo console](https://console.choreo.dev) using your credentials.
- Click on the user profile in the top right corner of the screen and select **Settings**.

![Settings Menu]({{base_path}}/assets/img/observe/settings-menu.png)

- If you are a member of multiple organizations select the appropriate organization from the top left-hand corner. For

>   more information on handling organizations please visit [here]({{base_path}}/observe/api-manager-analytics/configure-analytics/working-with-organizations).

Reviewer: ```suggestion
  more information on handling organizations, see [here]({{base_path}}/observe/api-manager-analytics/configure-analytics/working-with-organizations).
```

![Org Selector]({{base_path}}/assets/img/observe/organization-selector.png)

- Choose the **On-prem Keys** tab and click on **Generate keys**.

![On-prem Key]({{base_path}}/assets/img/observe/on-prem-key.png)

- Provide a suitable name for your environment (e.g., customer1-dev) and click the **Generate** button.
- You will be provided with a key in the next step. Copy that key before closing the dialog box.

- Now you can use this key to configure the Gateway to publish stats into the Analytics cloud. Follow [this
  document]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) if you are using
  API Gateway and follow
  [this document]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-microgateway)
  if you are using Choreo Connect Gateway.
  
    !!! info

>         **Free users** without a valid subscription for Choreo, will be limited to generate only 3 keys which will be valid 

Reviewer: ```suggestion
        **Free users** without a valid subscription for Choreo, will be limited to generate only 3 keys that will be valid 
```

>         for **2 weeks**. The validity of the keys can be temporarily extended by visiting

Reviewer: ```suggestion
        for **2 weeks**. The validity of the keys can be temporarily extended by
```

>          [contact sales](https://wso2.com/contact/) and submitting a request.

Reviewer: ```suggestion
         [contacting sales](https://wso2.com/contact/) and submitting a request.
```
